code/modules/mob/living/emote.dm

```diff
@@ -110,6 +110,9 @@
 	. = ..()
 	if(. && isliving(user))
 		var/mob/living/L = user
+		if(L.status_flags & GODMODE) //this will permasleep them due to how GODMODE works, I have never in all my days seen why anyone would want this
+			to_chat(user, "<span class='warning'>Your current invincibility also makes you invincible to waking up, ever. Like, forever and ever. Seriously. So let's not faint, OK?</span>")
```

Comment:
wait maybe I was wrong and the correct word actually is serious

I don't know anymore ahhhhhhhhhhhhhh

Reply:
It doesn't really matter in this situation